### PR TITLE
DEVDOCS-2773: Updated thumbnail property

### PR DESCRIPTION
**Pull request by ektravel**

### CODE REVIEW

File: reference/widgets.v3.yml [hunk 1]
```diff
@@ -1231,11 +1231,19 @@ definitions:
...
+          type:
+            type: string
+            example: image
+          valueKey:
+            type: string
+            example: imageUrl.src
       schema:
         type: array
         description: The schema used for each element in the array.
+        items: {}
```
Comment (bc-jcha) [inline reference/widgets.v3.yml:1246]: I think this is `widgetSchemaTab|widgetSchemaArray`

Comment (ektravel) [reply reference/widgets.v3.yml:1246]: @bc-jcha This is `widgetSchemaArray`. The `items` keyword is required in arrays per [Swagger documentation](https://swagger.io/docs/specification/data-models/data-types/#array). The value of `items` is a schema that describes the type and format of array items.

Comment (bc-jcha) [reply reference/widgets.v3.yml:1246]: Per our offline discussion, api-specs repo already has the items empty, so we will proceed with the current code for now.

**Review by bc-jcha — APPROVED** [review]

File: reference/widgets.v3.yml (hunk 1 above)
Comment (aglensmith) [reply reference/widgets.v3.yml:1246]: Merging, but we should specify the type for items.

**Pull request merged**